src/commands/balance.ts

```diff
@@ -24,8 +24,32 @@ const command: CommandInterface = {
       return;
     }
 
+    let addy: string;
+    try {
+      addy = await blockweave.wallets.jwkToAddress(wallet);
+    } catch (e) {
+      console.log(clc.red('Unable to decrypt wallet address.'));
+      if (debug) console.log(e);
+    }
+
+    if (useBundler) {
+      try {
+        const res = await bundler.get(`/account/balance?address=${addy}`);
```

Comment:
I noticed that this one to get the address is being called here and https://github.com/textury/arkb/pull/35/files#diff-0cb87263fc2cd13d57e280a254b560b49292258ea6134b4d08b1ba2ebf1eea63R48 , let's best use a single call that will be available in the bundler class here: https://github.com/textury/arkb/blob/01b9fecba930b65c7d758e017a723aa2fed87053/src/utils/bundler.ts